config/locales/en.yml

```diff
@@ -29,21 +29,6 @@
 
 en:
   app_name: 'Stanford Digital Repository'
-  activemodel:
-    errors:
-      models:
-        related_work:
-          attributes:
-            relationship:
-              inclusion: '%{value} is not a valid relationship for related works'
```

Comment:
No need for a custom message. The validation is only for documentation and could never be false.